doc/source/development/rfc/rfc102_embedded_resources.rst

```diff
@@ -0,0 +1,166 @@
+.. _rfc-102:
+
+===================================================================
+RFC 102: Embedding resource files into libgdal
+===================================================================
+
+============== =============================================
+Author:        Even Rouault
+Contact:       even.rouault @ spatialys.com
+Started:       2024-Oct-01
+Status:        Draft
+Target:        GDAL 3.11
+============== =============================================
+
+Summary
+-------
+
+This RFC uses C23 ``#embed`` pre-processor directive, when available,
+to be able to optionally embed GDAL resource files directly into libgdal.
+
+A similar `PROJ RFC-8 <https://github.com/OSGeo/PROJ/pull/4274>`__ has been
+submitted for PROJ to embed its :file:`proj.db` and :file:`proj.ini` files.
+
+Motivation
+----------
+
+Some parts of GDAL core, but mostly drivers, depend on a number of resource
+files for correct execution. Locating those resource files on the filesystem
```

Comment:
I find parts of this paragraph difficult to read. Does this convey the meaning of the first two sentences:

".. mostly drivers, require external resource files located in the filesystem. Locating these resource files is difficult for use cases where the GDAL binaries are relocated during installation time."

Take this with a grain of salt though, English isn't my first language.

Reply:
thanks, adopted